server/protocols/protocols-imap4/src/test/java/org/apache/james/imapserver/netty/IMAPServerTest.java

```diff
@@ -970,4 +977,106 @@ void searchingASingleUTF8CriterionShouldComplete() throws Exception {
             store.close();
         }
     }
+
+    @Nested
+    class Idle {
+        IMAPServer imapServer;
+        private int port;
+        private MailboxSession mailboxSession;
+        private MessageManager inbox;
+
+        @BeforeEach
+        void beforeEach() throws Exception {
+            imapServer = createImapServer("imapServer.xml");
+            port = imapServer.getListenAddresses().get(0).getPort();
+            mailboxSession = memoryIntegrationResources.getMailboxManager().createSystemSession(USER);
+            memoryIntegrationResources.getMailboxManager()
+                .createMailbox(MailboxPath.inbox(USER), mailboxSession);
+            inbox = memoryIntegrationResources.getMailboxManager().getMailbox(MailboxPath.inbox(USER), mailboxSession);
+        }
+
+        @AfterEach
+        void tearDown() {
+            imapServer.destroy();
+        }
+
+        @Test
+        void idleShouldSendInitialContinuation() throws Exception {
+            SocketChannel server = SocketChannel.open();
```

Comment:
This is more like a client than a server? BTW should we put this SocketChannel.open() into BeforeEach and close it in AfterEach?

Reply:
> This is more like a client than a server?

Yes we are opening a TCP client connection to test the server.

> BTW should we put this SocketChannel.open() into BeforeEach and close it in AfterEach?

:+1: